cli/formatter/formatter.go

```diff
@@ -0,0 +1,57 @@
+package formatter
+
+import (
+	"sort"
+	"strings"
+)
+
+//Format key to output
+const (
+	TableFormat = "table"
+	RawFormat   = "raw"
+)
+
+//IsTable to verify if table or raw
+func IsTable(format string) bool {
+	return strings.HasPrefix(format, TableFormat)
+}
+
+//PreFormat is to format the format option
+func PreFormat(format string) string {
+	if IsTable(format) {
+		format = format[len(TableFormat):]
+		//cut the space
+		format = strings.Trim(format, " ")
+		//input by cmd of "\t" is "\\t"
+		replace := strings.NewReplacer(`\t`, "\t", `\n`, "\n")
+		format = replace.Replace(format)
+
+		if format[len(format)-1:] != "\n" {
+			format += "\n"
+		}
+	} else {
+		format = `Name:{{.Name}}
+ID:{{.ID}}
+Status:{{.Status}}
+Created:{{.Created}}
+Image:{{.Image}}
+Runtime:{{.Runtime}}
+`
+	}
+	return format
+}
+
+//LabelsToString is to transfor the labels from map to string
+func LabelsToString(labels map[string]string) string {
+	var labelstring string
+	sortedkeys := make([]string, 0)
+	for key := range labels {
+		sortedkeys = append(sortedkeys, key)
+	}
+	sort.Strings(sortedkeys)
+	for _, key := range sortedkeys {
+		keyValueString := key + "=" + labels[key] + " "
```

Comment:
better to use `labelstring += fmt.Sprintf("%s = %s", key, value)`

Reply:
good suggestion.